patches/bullet.patch

```diff
@@ -0,0 +1,31 @@
+--- libs/bullet/CMakeLists.txt
++++ libs/bullet/CMakeLists.txt
+@@ -125,9 +125,11 @@ ENDIF(INTERNAL_CREATE_MSVC_RELATIVE_PATH_PROJECTFILES)
+ 
+ ENDIF (WIN32)
+ 
+-
+-OPTION(BUILD_CPU_DEMOS "Build original Bullet CPU examples"        ON)
+-
++IF (NOT ANDROID)
++	OPTION(BUILD_CPU_DEMOS "Build original Bullet CPU examples"        ON)
++ELSE ()
++        OPTION(BUILD_CPU_DEMOS "Don't build original Bullet CPU examples"  OFF)
```

Comment:
nit: The doc string should be the same in both cases, even if the default value is `OFF` on Android.